### ISSUE TITLE
Candidate Recommendation tracking issue

### ISSUE THREAD

**Issue by chrisn** [w3c/picture-in-picture#219]
This issue tracks the issues needed to prepare the Picture in Picture API for publication as Candidate Recommendation.

Open issues:

- [ ] [Should PiP video removed from the DOM leave PiP?](https://github.com/w3c/picture-in-picture/issues/99)
- [ ] [Who is responsible for the user interface for PiP?](https://github.com/w3c/picture-in-picture/issues/179)
- [ ] [Should include / require spoofing protections](https://github.com/w3c/picture-in-picture/issues/182)
- [ ] [disablePictureInPicture interoperability](https://github.com/w3c/picture-in-picture/issues/184)
- [ ] [Remove monkey patch sections](https://github.com/w3c/picture-in-picture/issues/191)
- [ ] [How to determine the correct video size?](https://github.com/w3c/picture-in-picture/issues/208)

Reviews and other tasks:

- [ ] Add Privacy Considerations
- [ ] Request horizontal reviews - see [HR status](https://docs.google.com/presentation/d/15KyxeFKx1V5VWbYLK2yELxQQbusfjgVXsADWZHOvIF8/edit#slide=id.g1273c919b8a_0_109)

**Comment by smaug----** [comment]
Looks like most of the in-parallel vs schedule a task handling is broken in the spec. Most of the time the algorithms should schedule a task and not use in-parallel.

**Comment by theIDinside** [comment]
Given that we have collaborated on #245 (though a PR, not an issue), I think we need the same kind of overhaul (@smaug---- hints at what kinds of things need addressing there as well) for `document.exitPictureInPicture()`. I think the changes there, will be less involved though.

So `exitPictureInPicture()` issue should be probably be added (https://github.com/w3c/picture-in-picture/issues/252).

We also need to resolve #250; as this will need changes in secondary specifications (unless we change tests to instead actually pause, upon being removed).

tldr;
- #245 - requestPictureInPicture()
- #252 - `document.exitPictureInPicture()` needs work, some of which is covered by #233, though I am not sure if it's complete. I can write a PR for exit as well, if that's desired @marcoscaceres || @chrisn ?
- #250 - Should removal of \<video\> element pause or not. The media spec says it should, yet we have WPT for PIP that expect it should not

**Comment by chrisn** [comment]
>  I can write a PR for exit as well, if that's desired @marcoscaceres || @chrisn ?

Thanks, that would be great. That PR and #245 I think then supersede #233.
